server/socketCommands/types.ts

```diff
@@ -174,6 +181,44 @@ export class Game {
     this.revealedLettersSet = new Set()
     this.elapsedSeconds = 0
   }
+
```

Comment:
leave a comment in the nextphase function to refactor later, there must be a way to make it look better